### ISSUE TITLE
[V1] gatsby-plugin-sitemap V2 doesn't build when using Gatsby version 1

### ISSUE THREAD

**Issue by kempsterrrr** [gatsbyjs/gatsby#9893]
## Description

I'm running a v1 version of Gatsby for an existing personal site and realised I hadn't submitted a sitemap to google search console so added gatsby-plugin-sitemap from the latest version on npm which kept failing with the following err.

`Error: Module not found: Error: Cannot resolve module 'gatsby' in /opt/build/r  epo/node_modules/gatsby-plugin-sitemap`
 
### Suggested resolution

Add a caveat on the readme.md for the plugin that should you be using V1 of gatsby you need to pull down the V1 of the plugin like so `yarn add gatsby-plugin-sitemap@1.2.25` or `npm install --save gatsby-plugin-sitemap@1.2.25`

### Steps to reproduce

To reproduce the issue just try to add gatsby-plugin-sitemap to a V1 gatsby site which hasn't been upgraded and run a build.

### Expected result

The site should build successfully and add a sitemap to stated output location in gatsby-config.s

### Actual result

Build failed locally and remote with the error `Error: Module not found: Error: Cannot resolve module 'gatsby' in /opt/build/r  epo/node_modules/gatsby-plugin-sitemap`

### Environment

```
System:
    OS: macOS 10.14
    CPU: x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 10.11.0 - /usr/local/bin/node
    Yarn: 1.5.1 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  Browsers:
    Chrome: 70.0.3538.77
    Safari: 12.0
  npmPackages:
    gatsby: ^1.9.250 => 1.9.269
    gatsby-link: ^1.6.39 => 1.6.44
    gatsby-plugin-feed: ^1.3.20 => 1.3.24
    gatsby-plugin-google-analytics: ^1.0.31 => 1.0.31
    gatsby-plugin-offline: ^1.0.15 => 1.0.18
    gatsby-plugin-react-helmet: ^1.0.8 => 1.0.8
    gatsby-plugin-sharp: ^1.6.42 => 1.6.46
    gatsby-plugin-sitemap: 1.2.25 => 1.2.25
    gatsby-plugin-typography: ^1.7.18 => 1.7.18
    gatsby-remark-copy-linked-files: ^1.5.31 => 1.5.35
    gatsby-remark-images: ^1.5.61 => 1.5.65
    gatsby-remark-prismjs: ^1.2.24 => 1.2.24
    gatsby-remark-responsive-iframe: ^1.4.18 => 1.4.19
    gatsby-remark-smartypants: ^1.4.12 => 1.4.12
    gatsby-source-filesystem: ^1.5.31 => 1.5.38
    gatsby-transformer-remark: ^1.7.41 => 1.7.44
    gatsby-transformer-sharp: ^1.6.22 => 1.6.26
  npmGlobalPackages:
    gatsby-cli: 2.4.2

```

**Comment by kempsterrrr** [comment]
Sorry to be clear. The environment details shared above were after I fixed the issue by installing the 1.2.25 version of `gatsby-plugin-sitemap` 

Hope that's OK

**Comment by jonniebigodes** [comment]
Glad you solved it. If you don't mind i'm closing this issue. If any other issues arise please create a new one.
